### PR TITLE
Migrate to django-prices 2.1

### LINKED ISSUE #4337: Upgrade django prices
With this PR: https://github.com/mirumee/django-prices/pull/73, django-prices will be able to support multi currencies. We should proceed with all described steps needed to upgrade package

### LINKED ISSUE #4378: Model with 'ManyToManyField' and 'TaxedMoneyField' raise error
### Steps to reproduce the problem
1.  Clone saleor (branch master)
2. install requirements (pipenv install)
3. docker-compose build
4. docker-compose up
5. add two fields saleor/order/models.py Order
i.e. 
- sped_label = models.FileField(upload_to="sped_label", blank=True)
- xaxa = models.CharField(max_length=255, default="", blank=True)
6. run manage.py
i.e. ./manage.py migrate

### Traceback
```─➤  ./manage.py migrate
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/base.py", line 361, in execute
    self.check()
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 65, in _run_checks
    issues.extend(super()._run_checks(**kwargs))
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/checks/urls.py", line 40, in check_url_namespaces_unique
    all_namespaces = _load_all_namespaces(resolver)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/core/checks/urls.py", line 57, in _load_all_namespaces
    url_patterns = getattr(resolver, 'url_patterns', [])
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/urls/resolvers.py", line 579, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/django/urls/resolvers.py", line 572, in urlconf_module
    return import_module(self.urlconf_name)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/urls.py", line 16, in <module>
    from .graphql.api import schema
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/graphql/api.py", line 3, in <module>
    from .account.schema import AccountMutations, AccountQueries
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/graphql/account/schema.py", line 6, in <module>
    from ..descriptions import DESCRIPTIONS
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/graphql/descriptions.py", line 4, in <module>
    from .order.resolvers import ORDER_SEARCH_FIELDS
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/graphql/order/resolvers.py", line 10, in <module>
    from .types import Order
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/graphql/order/types.py", line 255, in <module>
    class Order(CountableDjangoObjectType):
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/graphene/utils/subclass_with_meta.py", line 52, in __init_subclass__
    super_class.__init_subclass_with_meta__(**options)
  File "/Users/kswiatek/workspace/saleor/saleor/saleor/graphql/core/connection.py", line 56, in __init_subclass_with_meta__
    super().__init_subclass_with_meta__(*args, connection=countable_conn, **kwargs)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/graphene_django/types.py", line 83, in __init_subclass_with_meta__
    construct_fields(model, registry, only_fields, exclude_fields), _as=Field
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/graphene_django/types.py", line 22, in construct_fields
    _model_fields = get_model_fields(model)
  File "/Users/kswiatek/.pyenv/versions/3.6.7/envs/saleor/lib/python3.6/site-packages/graphene_django/utils/utils.py", line 39, in get_model_fields
    list(model._meta.fields) + list(model._meta.local_many_to_many)
TypeError: '<' not supported between instances of 'ManyToManyField' and 'TaxedMoneyField'
```

### LINKED ISSUE #4636: No date picker in the voucher forms of dashboard 1.0
The start and end dates are a raw text input. Instead we should render as a simple date field (instead of datetime). That way, we keep a backward compatibility.

A fix is available in a WiP.

### PR DESCRIPTION
**Pull request by NyanKiyoshi**
Closes #4337, fixes #4636, fixes #4378.

## TBD
- [x] Rebase
- [x] https://github.com/mirumee/django-prices/pull/90;
- [x] Release of django-prices 2.1

### Dashboard 1.0
- [x] Update product form
- [x] Update product variant form

### GraphQL
- [x] Fix accidental refactor of inputs
- [x] Deprecate invalid naming in the graphql types

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] Database migration files are up to date.
1. [x] The changes are tested.
1. [x] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: saleor/graphql/schema.graphql [hunk 1]
```diff
@@ -403,7 +403,7 @@ type Checkout implements Node {
   shippingAddress: Address
   shippingMethod: ShippingMethod
   note: String!
-  discountAmount: Money
+  discountAmount: Float!
```
Comment (NyanKiyoshi) [inline saleor/graphql/schema.graphql]: To deprecate and return money

File: Pipfile [hunk 2]
```diff
@@ -23,7 +23,7 @@ django-impersonate = ">=1.3"
 django-measurement = "==3.0.0"
 django-mptt = ">=0.7.1"
 django-phonenumber-field = ">=1.3.0,<3.0"
-django-prices = ">=1.0.2,<2.0.0"
+django-prices = ">=2.0.0,<3.0.0"
```
Comment (NyanKiyoshi) [inline Pipfile]: >=2.1

File: saleor/dashboard/widgets.py [hunk 3]
```diff
@@ -17,10 +16,15 @@ def __init__(self, attrs=None):
 class MoneyRangeWidget(widgets.RangeWidget):
     def __init__(self, attrs=None):
         self.currency = getattr(settings, "DEFAULT_CURRENCY")
-        money_widgets = (MoneyInput(self.currency), MoneyInput(self.currency))
+        money_widgets = (forms.NumberInput(), forms.NumberInput())
         # pylint: disable=bad-super-call
         super(widgets.RangeWidget, self).__init__(money_widgets, attrs)
 
+    def decompress(self, value):
```
Comment (NyanKiyoshi) [inline saleor/dashboard/widgets.py]: unrelated?

File: saleor/giftcard/migrations/0002_auto_20190814_0413.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        ),
+        migrations.RenameField(
+            model_name="giftcard",
+            old_name="initial_balance",
+            new_name="initial_balance_amount",
+        ),
+        migrations.AddField(
+            model_name="giftcard",
+            name="currency",
+            field=models.CharField(default="USD", max_length=3),
```
Comment (NyanKiyoshi) [inline saleor/giftcard/migrations/0002_auto_20190814_0413.py]: Should be taken from the settings

File: saleor/order/models.py [hunk 5]
```diff
@@ -81,59 +81,82 @@ class Order(models.Model):
...
     # Token of a checkout instance that this order was created from
     checkout_token = models.CharField(max_length=36, blank=True)
-    total_net = MoneyField(
-        currency=settings.DEFAULT_CURRENCY,
+
+    total_net_amount = models.DecimalField(
         max_digits=settings.DEFAULT_MAX_DIGITS,
         decimal_places=settings.DEFAULT_DECIMAL_PLACES,
-        default=zero_money,
+        default="0",
```
Comment (NyanKiyoshi) [inline saleor/order/models.py]: For consistency: `0`

File: saleor/shipping/migrations/0017_auto_20190814_0413.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+            model_name="shippingmethod", old_name="price", new_name="price_amount"
+        ),
+        migrations.AddField(
+            model_name="shippingmethod",
+            name="currency",
+            field=models.CharField(default="USD", max_length=10),
+        ),
+        migrations.AddField(
+            model_name="shippingmethod",
+            name="external_id",
```
Comment (NyanKiyoshi) [inline saleor/shipping/migrations/0017_auto_20190814_0413.py]: Were is this coming form?

File: saleor/shipping/migrations/0017_auto_20190814_0413.py [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+            field=models.CharField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AddField(
+            model_name="shippingmethod",
+            name="maximum_order_price_currency",
```
Comment (NyanKiyoshi) [inline saleor/shipping/migrations/0017_auto_20190814_0413.py]: Should just be "currency"

**Comment by django-queries** [comment]
Here is the report for bb87c0ca5a267b2f76714a8837a9ba04d1c48f0d (NyanKiyoshi/saleor @ django-prices/migrate/2.0)
Base comparison is b76eaeedff0e9996a119fd537aada7e45685585d.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  add billing address to checkout     	         34	         34	             20
  add shipping to checkout            	          7	          7	              0
  checkout payment charge             	         14	         14	              0
  complete checkout                   	          6	          6	              0
  create checkout                     	         48	         48	             24

# api.benchmark homepage
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  retrieve main menu                  	          5	          5	              0
  retrieve product list               	          4	          4	              0
  retrieve secondary menu             	          5	          5	              0
  retrieve shop                       	          2	          2	              0

# api.benchmark product
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  product details                     	         13	         13	              3

# api.benchmark variant
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  retrieve variant list               	         15	         15	              8
```

</p>
</details>

File: saleor/dashboard/product/forms.py [hunk 8]
```diff
@@ -323,14 +343,22 @@ def clean_seo_description(self):
     def save(self, commit=True):
         attributes = self.get_saved_attributes()
         self.instance.attributes = attributes
+
+        price = self.cleaned_data["price"]
+        if not price.currency:
```
Comment (Kwaidan00) [inline saleor/dashboard/product/forms.py]: How it could happen that price has `None` as currency?

File: saleor/discount/migrations/0017_auto_20190814_0413.py [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    operations = [
+        migrations.RenameField(
+            model_name="voucher",
+            old_name="min_amount_spent",
+            new_name="min_spent_amount",
+        ),
+        migrations.AddField(
+            model_name="voucher",
+            name="currency",
+            field=models.CharField(default=settings.DEFAULT_CURRENCY, max_length=10),
```
Comment (Kwaidan00) [inline saleor/discount/migrations/0017_auto_20190814_0413.py]: In saleor/checkout/migrations/0021_checkout_currency.py `currency` field has `max_length=3`, here is `max_length=10`. I think should we use one value for max length.

File: saleor/extensions/plugins/avatax/__init__.py [hunk 10]
```diff
@@ -255,16 +255,16 @@ def get_order_lines_data(order: "Order") -> List[Dict[str, str]]:
...
             data=data,
             quantity=line.quantity,
-            amount=line.unit_price_net.amount * line.quantity,
+            amount=line.unit_price_net_amount * line.quantity,
             tax_code=tax_code,
             item_code=line.variant.sku,
             description=line.variant.product.plain_text_description,
         )
-    if order.discount_amount and order.discount_amount.amount:
+    if order.discount_amount and order.discount_amount:
```
Comment (Kwaidan00) [inline saleor/extensions/plugins/avatax/__init__.py]: If `p and p`? :)

File: saleor/extensions/plugins/avatax/plugin.py [hunk 11]
```diff
@@ -154,7 +154,7 @@ def calculate_checkout_total(
         total_gross = Money(amount=total_net + tax, currency=currency)
         total_net = Money(amount=total_net, currency=currency)
         total = TaxedMoney(net=total_net, gross=total_gross)
-        voucher_amount = checkout.discount_amount
+        voucher_amount = checkout.discount
```
Comment (Kwaidan00) [inline saleor/extensions/plugins/avatax/plugin.py]: `checkout.discount` is `Money` instance; I think the variable name `voucher_amount` will be **very** confusing in the future.
Maybe it should be `voucher_value` or something else?

File: saleor/checkout/models.py [hunk 12]
```diff
@@ -112,19 +117,17 @@ def get_subtotal(self, discounts=None):
 
     def get_total(self, discounts=None):
         """Return the total cost of the checkout."""
-        total = (
-            self.get_subtotal(discounts)
-            + self.get_shipping_price()
-            - self.discount_amount
-        )
+        total = self.get_subtotal(discounts) + self.get_shipping_price() - self.discount
```
Comment (Kwaidan00) [inline saleor/checkout/models.py]: I think `get_subtotal` should use `zero_money(currency=self.currency)` instead of `DEFAULT_CURRENCY`.

File: saleor/graphql/checkout/types.py [hunk 13]
```diff
@@ -165,9 +167,13 @@ def resolve_is_shipping_required(root: models.Checkout, _info):
 
     @staticmethod
     @permission_required("order.manage_orders")
-    def resolve_private_meta(root, _info):
+    def resolve_private_meta(root: models.Checkout, _info):
```
Comment (Kwaidan00) [inline saleor/graphql/checkout/types.py]: Every type annotation is nice :) 👍

File: saleor/graphql/discount/types.py [hunk 14]
```diff
@@ -102,6 +102,9 @@ class Voucher(CountableDjangoObjectType):
         required=True,
     )
     type = VoucherTypeEnum(description="Determines a type of voucher", required=True)
+    min_amount_spent = graphene.Field(
+        types.Money, deprecation_reason="Use the minAmount field instead."
```
Comment (Kwaidan00) [inline saleor/graphql/discount/types.py]: Shouldn't it be `use minSpent field instead`?

File: saleor/graphql/schema.graphql [hunk 15]
```diff
@@ -2041,6 +2042,7 @@ type Order implements Node {
   totalBalance: Money!
   userEmail: String
   isShippingRequired: Boolean!
+  discountAmount: Money!
```
Comment (Kwaidan00) [inline saleor/graphql/schema.graphql]: Why `discountAmount` is not deprecated, but `checkout.discountAmount` is?

File: saleor/order/utils.py [hunk 16]
```diff
@@ -111,17 +111,17 @@ def decorator(*args, **kwargs):
         if kwargs.pop("update_voucher_discount", True):
             order = args[0]
             try:
-                discount_amount = get_voucher_discount_for_order(order)
+                discount = get_voucher_discount_for_order(order)
             except NotApplicable:
-                discount_amount = zero_money()
-            order.discount_amount = discount_amount
+                discount = zero_money()
```
Comment (Kwaidan00) [inline saleor/order/utils.py]: I suggest `zero_money(currency=order.currency)`

File: saleor/product/migrations/0105_auto_20190814_0413.py [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+        ),
+        migrations.RenameField(
+            model_name="productvariant",
+            old_name="price_override",
+            new_name="price_override_amount",
+        ),
+        migrations.AddField(
+            model_name="product",
+            name="currency",
+            field=models.CharField(default=settings.DEFAULT_CURRENCY, max_length=10),
```
Comment (Kwaidan00) [inline saleor/product/migrations/0105_auto_20190814_0413.py]: Again: `max_length=10`

File: saleor/shipping/migrations/0017_auto_20190814_0413.py [hunk 18]
```diff
@@ -0,0 +1,30 @@
...
+            old_name="minimum_order_price",
+            new_name="minimum_order_price_amount",
+        ),
+        migrations.RenameField(
+            model_name="shippingmethod", old_name="price", new_name="price_amount"
+        ),
+        migrations.AddField(
+            model_name="shippingmethod",
+            name="currency",
+            field=models.CharField(default=settings.DEFAULT_CURRENCY, max_length=10),
```
Comment (Kwaidan00) [inline saleor/shipping/migrations/0017_auto_20190814_0413.py]: Again, `max_length=10`

File: saleor/dashboard/product/forms.py (hunk 8 above)
Comment (NyanKiyoshi) [reply saleor/dashboard/product/forms.py]: Not sure. It happened a few times in the dashboard. I will try to dig around to find a consistent way of reproducing it.

**Comment by NyanKiyoshi** [comment]
A lot of the money logic really needs refactoring as it was never meant to be able to handle multiple currencies. Overall, it should be able to handle having different currencies, e.g. the storefront owner changed the default currency. But it is not meant to be achievable.

For now we only want to upgrade to the latest django-prices to fix a blocking bug (#4378) and once we get the time for it, we will implement proper multi-currency support, which shouldn't depend on the settings' default currency.

**Comment by Kwaidan00** [comment]
@NyanKiyoshi you're right, a lot of code will have to be refactored for multi-currency support. I think the hardest part will be sorting and filtering stuff. And of course checking if values in sum has the same currency - or catching exception.

File: saleor/dashboard/widgets.py [hunk 19]
```diff
@@ -17,7 +16,7 @@ def __init__(self, attrs=None):
 class MoneyRangeWidget(widgets.RangeWidget):
     def __init__(self, attrs=None):
         self.currency = getattr(settings, "DEFAULT_CURRENCY")
```
Comment (korycins) [inline saleor/dashboard/widgets.py]: Is it expected that you take currency from settings here?

Comment (NyanKiyoshi) [reply saleor/dashboard/widgets.py]: The issue here is that we don't really know what currency we want to filter the objects by. So we assume it is the default currency until we drop dashboard 1.0 and implement multi-currency support.

File: saleor/dashboard/product/forms.py (hunk 8 above)
Comment (maarcingebala) [reply saleor/dashboard/product/forms.py]: Right, his might be an issue somewhere else that is propagating to forms.

File: saleor/graphql/order/types.py [hunk 20]
```diff
@@ -138,6 +138,10 @@ def resolve_fulfilled_items(root: models.OrderEvent, _info):
         lines = root.parameters.get("fulfilled_items", None)
         return models.FulfillmentLine.objects.filter(pk__in=lines)
 
+    @staticmethod
+    def resolve_discount_amount(root: models.Order, _info):
```
Comment (maarcingebala) [inline saleor/graphql/order/types.py]: Does an `OrderEvent` have the discount field? I don't see it in the schema.

File: saleor/graphql/shipping/mutations.py [hunk 21]
```diff
@@ -11,11 +11,29 @@
 
 class ShippingPriceInput(graphene.InputObjectType):
     name = graphene.String(description="Name of the shipping method.")
-    price = Decimal(description="Shipping price of the shipping method.")
+    price = Decimal(
+        description=(
+            "Deprecated: use priceAmount instead.\n"
```
Comment (maarcingebala) [inline saleor/graphql/shipping/mutations.py]: Can't we keep using the `price` field here? I mean it seems the old name seems more user-friendly and it's still the same type `Decimal` with the new one. Or am I missing something?

File: saleor/dashboard/shipping/forms.py [hunk 22]
```diff
@@ -183,6 +217,15 @@ def clean(self):
             )
         return data
 
+    def save(self, commit=True):
+        for k, v in self.cleaned_data.items():
+            if not isinstance(v, Money):
+                continue
+            if not v.currency:
+                v.currency = settings.DEFAULT_CURRENCY
```
Comment (maarcingebala) [inline saleor/dashboard/shipping/forms.py]: Couldn't this be handled by the `MoneyField` underneath?

File: templates/checkout/shipping_method.html [hunk 23]
```diff
@@ -12,7 +12,7 @@ <h3>{% trans "Shipping address" context "Checkout shipping address title" %}</h3
     </div>
     <p>
       <a class="text-uppercase" href="{% url 'checkout:shipping-address' %}">
-        {% trans "Select other address" %}
+        {% trans "Select another address" %}
```
[the remark this answers is not in the document]
Comment (timcla) [reply templates/checkout/shipping_method.html]: 'Other' if you already have two addresses written into the system and you want to switch to the second one.

'Another' if you want to add an additional address that is not in the system yet

File: saleor/dashboard/shipping/forms.py (hunk 22 above)
Comment (NyanKiyoshi) [reply saleor/dashboard/shipping/forms.py]: It should be but there is sometimes it just return `Money("1.23", "")`. I need to dig this, probably a bug on `django-prices`.

c.f. https://github.com/mirumee/saleor/pull/4639#discussion_r314691871

File: templates/checkout/shipping_method.html (hunk 23 above)
[the remark this answers is not in the document]
Comment (timcla) [reply templates/checkout/shipping_method.html]: > `other` sounds to me like "select the other address". Not sure 😉 Will ask.

Can be 'Select other address' or 'select the other address'

First one is fine from context, though less grammatically correct, and actually sounds better.

[the remark this answers is not in the document]
Comment (NyanKiyoshi) [reply templates/checkout/shipping_method.html]: Thanks 🎉

File: saleor/graphql/shipping/mutations.py (hunk 21 above)
Comment (NyanKiyoshi) [reply saleor/graphql/shipping/mutations.py]: I think it's fine to keep it as `price: decimal` then. Would be great if we could have a `to_field` attribute in the `Scalar` object to have no need for custom logic.

File: saleor/dashboard/forms.py [hunk 24]
```diff
@@ -1,8 +1,50 @@
...
+            set_initial_money(self.instance, field_name, field)
+
+    def save(self, commit=True):
+        for field_name, field in self._money_fields.items():
+            value = self.cleaned_data[field_name]  # type: Money
+            if value is None:
+                # Preserve the currency value
+                setattr(self.instance, f"{field_name}_amount", None)
+            else:
+                print(field_name, field_name, value)
```
Comment (maarcingebala) [inline saleor/dashboard/forms.py]: 👀
